lib/endpoints/class-wp-rest-revisions-controller.php

```diff
@@ -386,7 +386,8 @@ public function get_collection_params() {
 	/**
 	 * Check the post excerpt and prepare it for single post output.
 	 *
-	 * @param string       $excerpt
+	 * @param string      $excerpt The post excerpt.
+	 * @param int|WP_Post $post    Post revision object.
```

Comment:
I don't believe the `int` is relevant here -- `get_the_excerpt()` function takes an `int|WP_Post`, but then uses `get_post` to convert it to a WP_Post before passing to the `get_the_excerpt` filter. The $post parameter documented for the filter itself is `WP_Post $post Post object.`, and since we don't do any conversion in this method I believe we'd want to avoid taking in a post ID integer.